client/landing/jetpack-cloud/sections/backups/main.jsx

```diff
@@ -8,17 +8,34 @@ import { connect } from 'react-redux';
  * Internal dependencies
  */
 import { getSelectedSiteId } from 'state/ui/selectors';
+import { requestActivityLogs } from 'state/data-getters';
 
 class BackupsPage extends Component {
 	render() {
-		return <div>Welcome to the backup detail page for site { this.props.siteId }</div>;
+		const { logs, siteId } = this.props;
+
+		const backupPoints = logs.filter( activity => {
+			return activity.activityIsRewindable === true;
+		} );
+
+		const list = backupPoints.map( point => <div>{ point.activityDate }</div> );
+
+		return (
+			<div>
+				<div>Welcome to the backup detail page for site { siteId }.</div>
+				<div>This site has { backupPoints.length } backups.</div>
+				<div>{ list }</div>
+			</div>
+		);
 	}
 }
 
 export default connect( state => {
 	const siteId = getSelectedSiteId( state );
+	const logs = siteId && requestActivityLogs( siteId, { group: 'rewind' } );
 
 	return {
 		siteId,
+		logs: logs?.data ?? [],
```

Comment:
This is interesting! What operator is `?.`, I haven't seen it before?

Reply:
It's the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) - a new handy operator for getting a deeply nested object's property. Basically, you don't have to check each chained property for existence.